Admin/Admin.php

```diff
@@ -1620,20 +1653,6 @@ public function setFormGroups(array $formGroups)
     }
 
     /**
-     * {@inheritdoc}
-     */
-    public function removeFieldFromFormGroup($key)
```

Comment:
Yes in theory it can be called from an extension as its public, but I don't see what would be the use case ?

If you really want to keep it, maybe I can keep an empty method in the ``Admin`` class only ?